test/test.gyp

```diff
@@ -190,6 +191,10 @@
             '_XOPEN_SOURCE=500',
           ],
         }],
+        [ 'OS == "zos"', {
+          'cflags': [ '-qlanglvl=stdc99', '-qxplink' ],
+          'ldflags': [ '-qxplink' ],
+        }],
```

Comment:
Does the test build not pick up/inherit `common.gypi`?
https://github.com/libuv/libuv/blob/c51ead5c704cae957b9323feef97e842e1bdb033/common.gypi#L206-L209